lib/tasks/server/middleware/serve-files/index.js

```diff
@@ -25,7 +25,7 @@ ServeFilesAddon.prototype.serverMiddleware = function(options) {
     autoIndex: false // disable directory listings
   });
 
-  var baseURL = cleanBaseURL(options.rootURL || options.baseURL);
+  var baseURL = options.rootURL === '' ? '/' : cleanBaseURL(options.rootURL || options.baseURL);
```

Comment:
is that a problem? `baseURL` is deprecated and shouldn't be used and especially not in combination with `rootURL`